### PR TITLE
Final UG for v1.4

### PR DESCRIPTION
**Pull request by owen-yap**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -124,7 +121,7 @@ Here is a quick summary of each GUI component within Docedex.
 | **Footer**            | Shows the location of the Docedex storage.                                                                                                                                                                                                                                  |
 
 
-### Notes about the GUI
+### Notes about the GUI (Please Read!)
```
Comment (praveenkrishna0512) [inline docs/UserGuide.md:124]: What's with the "Please Read" XD

File: docs/UserGuide.md [hunk 2]
```diff
@@ -61,13 +61,10 @@ than current GUI apps in the industry.
 
 ---
 
-## **Glossary**
-
-Here are some descriptions of the words we use throughout the User Guide:
-- Parameter: Parameters are like fields in a form you are required to fill up. They include details you would include about the doctor/patient.
-
```
Comment (praveenkrishna0512) [inline docs/UserGuide.md:68]: I believe we still need a glossary section, we shouldn't just remove it

File: docs/UserGuide.md [hunk 3]
```diff
@@ -262,7 +263,7 @@ For more information see: [Selecting doctors or patients through commands](#sele
     - The index refers to the index number shown in the displayed doctor list.
     - Users can edit specific doctors in the clinic by providing at least one of the optional parameters. Specified parameters will be updated to the input values and all other parameter values will remain the same.
     - At least one of the optional parameters must be provided.
-    - When editing tags, the existing tags of the person will be removed i.e., adding of tags is not cumulative. You can remove all the person’s tags by typing `t/` without any tag string.
+      - When editing tags, you have to include any previous tags that was already included in the doctor, or else these tags will be removed. This also means that you can use `t/` to remove all tags from a doctor using the `edit-doc` command.
```
Comment (praveenkrishna0512) [inline docs/UserGuide.md:266]: Do remove the whitespace at the start of this line! Else it will be indented

**Review by praveenkrishna0512 — CHANGES_REQUESTED** [review]
Minor changes to be done!

**Review by vigonometry — COMMENTED** [review]
Kindly make changes requested by @praveenkrishna0512 
Also, there are some conflicts to resolve.

Other than that LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (owen-yap) [reply docs/UserGuide.md:124]: Important bro

File: docs/UserGuide.md [hunk 4]
```diff
@@ -167,7 +156,8 @@ Here is a quick summary of each GUI component within Docedex.
 | **Information Card**  | Displays all information about a selected doctor or patient.                                                                                                                                                                                                                |
 | **Footer**            | Shows the location of the Docedex storage.                                                                                                                                                                                                                                  |
 
-### Notes about the GUI
+
+### Notes about the GUI (Please Read!)
```
Comment (praveenkrishna0512) [inline docs/UserGuide.md:160]: Could we revert this back, thanks!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -61,21 +61,10 @@ than current GUI apps in the industry.
...
-
-Here are some descriptions of the words we use throughout the User Guide:
-- **Parameter**: Parameters are like fields in a form you are required to fill up.
-They include details you would include about the doctor/patient.
-- **Command**: An input from the user that tells Docedex to perform an action (ie. add a doctor).
-- **GUI**: Graphical User Interface (GUI) represents the visual display of Docedex that users can see.
-- **GUI component**: A subsection of the Graphical User Interface. For more information on specific GUI
-components, refer to [this section](#navigating-the-graphical-user-interface-gui).
-- **CLI**: Command Line Interface (CLI) represents a text-based user interface to interact with the application.
-
```
Comment (praveenkrishna0512) [inline docs/UserGuide.md:76]: Just to clarify, is there a reason you do not want the glossary?

**Review by praveenkrishna0512 — CHANGES_REQUESTED** [review]
Requesting minor changes/clarifications before merging!

File: docs/UserGuide.md [hunk 6]
```diff
@@ -61,21 +61,10 @@ than current GUI apps in the industry.
...
 ### Parameter Information
```
Comment (praveenkrishna0512) [inline docs/UserGuide.md:64]:
```suggestion
## **Glossary**

### Definitions

Here are some descriptions of the words we use throughout the User Guide:
- **Parameter**: Parameters are like fields in a form you are required to fill up.
They include details you would include about the doctor/patient.
- **Command**: An input from the user that tells Docedex to perform an action (ie. add a doctor).
- **GUI**: Graphical User Interface (GUI) represents the visual display of Docedex that users can see.
- **GUI component**: A subsection of the Graphical User Interface. For more information on specific GUI
components, refer to [this section](#navigating-the-graphical-user-interface-gui).
- **CLI**: Command Line Interface (CLI) represents a text-based user interface to interact with the application.

### Parameter Information
```

**Review by praveenkrishna0512 — APPROVED** [review]
LGTM!

**Pull request merged**